pkg/kubectl/cmd/get/get.go

```diff
@@ -436,12 +441,12 @@ func (o *GetOptions) transformRequests(req *rest.Request) {
 
 // Run performs the get operation.
 // TODO: remove the need to pass these arguments, like other commands.
```

Comment:
nit: We could remove the TODO along with this commit.

Reply:
Thanks a lot for your always helpful review. I was thinking of doing this, but I came to the conclusion that there is still work to be done: getting rid of the `cmd` argument. This is a a more significant refactor, however, so I opted to leave that for another commit.